migrations/2022-06-13-124806_post_report_name_length/up.sql

```diff
@@ -0,0 +1,2 @@
+-- remove the length limit
+alter table post_report alter column original_post_name type text;
```

Comment:
Make this match the post column name exactly:

`alter table post report alter column original_post_name type varchar(200)`;

Reply:
I thought it would be better to remove the limit, because that will cause another problem if we ever increase the limit for post.name and forget to change it here, but okay.